### DISCUSSION TITLE
Should we release 0.73.1?

### DISCUSSION THREAD

**Discussion by huntie** [reactwg/react-native-releases#94] [Patches]
Current Release: [0.73.0](https://github.com/facebook/react-native/releases/tag/v0.73.0)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

**Comment by huntie** [comment]
To cherry pick (carried over from [Road to 0.73.0](https://github.com/reactwg/react-native-releases/discussions/64)):

- [x] https://github.com/facebook/react-native/pull/41657
- [x] https://github.com/facebook/react-native/pull/41787 (Plus corresponding bump of `debugger-frontend`) cc @byCedric
    - [x] https://github.com/facebook/react-native/pull/41840

**Comment by hurali97** [comment]
To cherry pick:

- https://github.com/facebook/react-native/pull/41781

**Reply by huntie** [reply]
Merged ✅

**Comment by byCedric** [comment]
Please pick:
- https://github.com/facebook/react-native/pull/41836

**Comment by robertying** [comment]
This PR https://github.com/facebook/react-native/pull/41875 fixes a config overwriting issue during `pod install`, which I believe is very likely to cause connection issues in production if one doesn't look too hard at the changes.

Hopefully the fix gets picked in this release as it's a non-breaking build-time bug fix.

**Reply by mikehardy** [reply]
Yeah - this one bit me! But it also made me realize I had a cloud-based (vs emulator-based) firebase functions test still in my test suite, so it had a useful side effect anyway ;-). Was definitely surprising to have my Info.plist changed during pod install though!

**Reply by Titozzz** [reply]
Picked! ✅

**Comment by szymonrybczak** [comment]
CLI Bump: https://github.com/facebook/react-native/pull/41882

**Reply by huntie** [reply]
@szymonrybczak Can you pick https://github.com/react-native-community/cli/pull/2198 across to the `12.x` branch in CLI and include this as `12.2.2`?

**Reply by szymonrybczak** [reply]
It is already :) https://github.com/react-native-community/cli/commit/0a3635c18cee5abf75824934fd2f8549c9d3a60b

**Reply by huntie** [reply]
Merged ✅

**Comment by kelset** [comment]
this looks important: https://github.com/facebook/react-native/pull/41875

**Comment by janicduplessis** [comment]
https://github.com/facebook/react-native/pull/41846 would be nice, however it is part of the @react-native/virtualized-lists package so I'm not sure how that works.

**Reply by huntie** [reply]
We can pick this ✅ It'll trigger a new minor release for that package (will be `0.73.4`).

**Reply by kelset** [reply]
IIRC it's basically following the same pattern as all the other versioned/released packages, they need to pick it and then there's a separate CI job that can be triggered to handle the release of it: https://reactnative.dev/contributing/release-updating-packages

**Reply by janicduplessis** [reply]
Awesome, thanks!

**Comment by jeveloper** [comment]
Gratefully asking to bump  the issue https://github.com/facebook/react-native/issues/41205.
I would say this is a quality of service as FlatLists are very commonly used and I recall bugs in RN 40s, 50s. 

Thank you kindly

**Comment by szymonrybczak** [comment]
CLI Bump https://github.com/facebook/react-native/pull/41927

**Reply by szymonrybczak** [reply]
cc. @huntie

**Reply by huntie** [reply]
Merged ✅

**Comment by tjzel** [comment]
Hey, could you pick https://github.com/facebook/react-native/commit/1a0e1741c0ab4f22b57a6a7ee86ff4287fa09103 ?

**Reply by huntie** [reply]
This will need to go in 0.73.2, will carry this request over.
